src/lib/core/datetime.c

```diff
@@ -68,6 +68,53 @@ datetime_strftime(const struct datetime *date, char *buf, size_t len,
 	return tnt_strftime(buf, len, fmt, &tm);
 }
 
+void
```

Comment:
You don't use this func outside of this .c file, so it should be static.

Reply:
Done. Thanks!